torch/utils/hipify/hipify_python.py

```diff
@@ -684,7 +693,7 @@ def repl(m):
                 or f.startswith("THCUNN/")
                 or (f.startswith("THC") and not f.startswith("THCP"))
             ):
-                return templ.format(get_hip_file_path(m.group(1)))
+                return templ.format(get_hip_file_path(m.group(1), False))
```

Comment:
@hgaspar Should this not be `return templ.format(get_hip_file_path(m.group(1), allow_same_dir))` instead?